integration-tests/rita.sh

```diff
@@ -74,4 +75,4 @@ else
   popd
 fi
 
-sudo -E python3 rita.py $@
+sudo -E PATH="$PATH:$HOME/.cargo/bin" python3 rita.py $@
```

Comment:
did you not add cargo to your own path? That should fix this problem too. I guess maybe we shouldn't assume people did that.

Reply:
@jkilpatr There were some hiccups with PATH passing in `sudo -E` on Travis during the A/B in CI ordeal, perhaps @nathande faced the same problem. I solved it for CI then by adding `cp <cargo_bin>/diesel /usr/bin/diesel` as a CI prep step. It is a bit problematic though because `~/.cargo/bin` being the cargo installation path is only a default and assuming that will be annoying to people with custom setups later on. On CI however we have control over that so the copying is sane there.

Reply:
@nathande Maybe make this into an env like `CARGO_BIN_PATH` defaulting to `~/.cargo/bin` which gets appended to `PATH` so that people with different setups can change that easily?